tests/draft4/future-keywords.json

```diff
@@ -0,0 +1,478 @@
+[
+    {
+        "description": "$anchor: location-independent identifier",
+        "schema": {
+            "allOf": [{
+                "$ref": "#foo"
+            }],
+            "$defs": {
+                "A": {
+                    "$anchor": "foo",
+                    "type": "integer"
+                }
+            }
+        },
+        "tests": [
+            {
+                "data": 1,
+                "description": "cannot match: $ref is not found",
+                "valid": false
```

Comment:
If an implementation doesn't know what `$anchor` is, then this isn't going to be `false`, it's going to emit an undefined reference error when it encounters `{$ref: "#foo}`

Reply:
agreed, I'll take this out. I wrote this before the discussion on undefined-behaviour-not-being-the-same-as-invalid.